<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSetRulesEditor</class>
 <widget class="QDialog" name="DataSetRulesEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataSetRulesEditor</string>
  </property>
  <widget class="QDialogButtonBox" name="buttons_">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="all_rules_">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="desc_">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Fields:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="fields_">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Highlight Fields</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="highlights_">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Equation</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="xero::scouting::views::EquationLineEdit" name="equation_">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="fore_">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Foreground</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Background</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>120</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QLabel" name="sample_">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sample</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="new_">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="matched_count_">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>(0 matched)</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="equ_error_">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="group_size_">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>75</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>(0 matched)</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>xero::scouting::views::EquationLineEdit</class>
   <extends>QLineEdit</extends>
   <header>EquationLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
